.travis.yml

```diff
@@ -11,13 +11,13 @@ matrix:
     env: TEST_TARGET=default
   - python: 3.5
     env: TEST_TARGET=default
-  - python: 3.5
-    env: TEST_TARGET=coding_standards
   - python: 3.6
     env: TEST_TARGET=default
-  allow_failures:
   - python: 3.5
     env: TEST_TARGET=coding_standards
+  allow_failures:
+  - python: 3.6
```

Comment:
This does not mean much but I like to run the coding standards with the latest Python available.